uaclient/entitlements/fips.py

```diff
@@ -43,6 +44,25 @@ class FIPSCommonEntitlement(repo.RepoEntitlement):
 
     help_doc_url = "https://ubuntu.com/security/certifications#fips"
 
+    def _check_for_xenial_cloud_instances(self):
```

Comment:
Let's name this method to describe what check we are performing. ++ Typing hints. And add a minor docstring.
```suggestion
    def _prevent_fips_on_xenial_cloud_instances(self) -> bool:
```